server/index.js

```diff
@@ -1,16 +1,22 @@
 const express = require('express');
 const app = express();
 const path = require('path');
+const axios = require('axios');
 
-app.use(express.static(__dirname + '../public/index.html'));
+app.use('/:listingId', express.static(__dirname + '../../public'));
 app.use(express.json());
 
-app.get('/:listingId', (req, res) => {
+app.get('/api/reservation/:listingId', (req, res) => {
   console.log('GET')
-  console.log('path: ', req.path)
-  const listingPage = path.join(__dirname, '../public/index.html')
-  res.sendFile(listingPage);
+  console.log('path: ', req.params.listingId)
+  axios.get(`http://localhost:5000/api/reservation/${req.params.listingId}`)
+    .then(data => {
+      console.log('data from reservation: ', data);
+      res.send(data.data)
+    })
+    .catch(err => {
+      console.log('failed to get reservation data for listing', err)
+    })
```

Comment:
Still a little confused what this endpoint does?

Reply:
when the bundle gets served and the App component mounts, it'll make a call to this endpoint to get the App's data.

Reply:
also might be refactored out after our most recent meeting

Reply:
gotcha gotcha.